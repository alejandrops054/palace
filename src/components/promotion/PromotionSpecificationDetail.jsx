/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from "react";
import { Button } from '../index'
import { Card, CardBody, CardTitle, Label, Input, Col, Row } from 'reactstrap';

function PromotionSpecificationDetail() {
    
    return (
        <>
            <Card>
                <CardBody>
                    <CardTitle tag="h5">Promotion Specification</CardTitle>
                    <Row>
                        <Col md={6}>
                            <Label>Specification Type</Label>
                            <Input type="select" name="select" id="exampleSelect">
                                <option>1</option>
                                <option>2</option>
                            </Input>
                        </Col>
                        <Col md={6}>
                            <Label>Status</Label>
                            <Input type="select" name="select" id="exampleSelect">
                                <option>1</option>
                                <option>2</option>
                            </Input>
                        </Col>
                    </Row>

                    <div className="d-flex justify-content-end">
                        <Button col={2}
                            typebootstrap="primary "
                            type="submit"
                            title="Save" />
                        <Button col={2}
                            typebootstrap="secondary"
                            type="button"
                            title="Cancel"
                        />
                    </div>
                </CardBody>
            </Card>
        </>
    );
}

export default PromotionSpecificationDetail;