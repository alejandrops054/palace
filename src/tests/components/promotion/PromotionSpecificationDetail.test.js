/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import PromotionSpecificationDetail from '../../../components/promotion/PromotionSpecificationDetail'

describe('PromotionSpecificationDetail component',() => {
    test('Debe mostrar snapshot', () => {
        
        const wrapper = shallow(<PromotionSpecificationDetail/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})