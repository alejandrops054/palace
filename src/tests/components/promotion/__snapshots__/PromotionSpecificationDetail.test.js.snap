// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PromotionSpecificationDetail component Debe mostrar snapshot 1`] = `
<Fragment>
  <Card
    tag="div"
  >
    <CardBody
      tag="div"
    >
      <CardTitle
        tag="h5"
      >
        Promotion Specification
      </CardTitle>
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          md={6}
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Specification Type
          </Label>
          <Input
            id="exampleSelect"
            name="select"
            type="select"
          >
            <option>
              1
            </option>
            <option>
              2
            </option>
          </Input>
        </Col>
        <Col
          md={6}
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            Status
          </Label>
          <Input
            id="exampleSelect"
            name="select"
            type="select"
          >
            <option>
              1
            </option>
            <option>
              2
            </option>
          </Input>
        </Col>
      </Row>
      <div
        className="d-flex justify-content-end"
      >
        <Button
          col={2}
          title="Save"
          type="submit"
          typebootstrap="primary "
        />
        <Button
          col={2}
          title="Cancel"
          type="button"
          typebootstrap="secondary"
        />
      </div>
    </CardBody>
  </Card>
</Fragment>
`;
